controllers/kustomization_controller.go

```diff
@@ -66,12 +66,6 @@ func (r *KustomizationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 
 	log := r.Log.WithValues(strings.ToLower(kustomization.Kind), req.NamespacedName)
 
-	kustomization = kustomizev1.KustomizationProgressing(kustomization)
-	if err := r.Status().Update(ctx, &kustomization); err != nil {
-		log.Error(err, "unable to update Kustomization status")
-		return ctrl.Result{Requeue: true}, err
-	}
-
 	if kustomization.Spec.Suspend {
 		msg := "Kustomization is suspended, skipping execution"
```

Comment:
Maybe

```suggestion
		msg := "Kustomization is suspended, skipping reconciliation"
```

Given the discussion we had about "sync" and "execution" being synonyms of "reconciliation" @stefanprodan .

Reply:
Can you please amend the commit with the above change

Reply:
As you please, done.